src/app/proto/validator/accounts/v2/web_api.ts

```diff
@@ -201,3 +201,18 @@ export interface ValidateKeystoresRequest {
    */
   keystoresPassword: string;
 }
+
+export interface RecoverWalletRequest {
+  /**
+   * 24 word Mnemonics
+   */
+  Mnemonic: string;
```

Comment:
Sorry I think when I mentioned these, they were capitalized in the google doc. However, we should be following camelCase like we do for other struct fields across the project

Reply:
I pushed to the pr that would resolve this issue. I did find it odd that this was not in camel case but I rolled with it